sched/scheduler/stateful_scheduler.go

```diff
@@ -476,6 +479,19 @@ func (s *statefulScheduler) updateStats() {
 			s.requestorsCounts[requestor][numWaitingTasksKey] += len(job.Tasks) - job.TasksCompleted - job.TasksRunning
 		}
 
+		if time.Now().Sub(job.TimeMarker) > LongJobDuration {
```

Comment:
Looks like TimeMarker never gets updated. Is that the intent? If so, why not just use TimeCreated?